### PR TITLE
Run all gating scenarios with want_sles12sp2=1

### PR DESCRIPTION
**Pull request by dirkmueller**
Just to be able to assess the state where we are with
doing the flip

### CODE REVIEW

**Comment by jsuchome** [comment]
Are those 2 jobs with nodenumber=4 supposed to deploy ceph?

**Comment by jsuchome** [comment]
+1

**Comment by toabctl** [comment]
+1

**Comment by dirkmueller** [comment]
@jsuchome well, its just a copy of the actual gating job with addition of want_sle12sp1, not sure what exactly it does..

**Comment by jsuchome** [comment]
OK, we'll see. For ceph deployments we might need to specifically set the os's (using want_node_os) if it isn't picked correctly automatically.

**Comment by rsalevsky** [comment]
+1

File: scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-sle12sp2.yaml [hunk 1]
```diff
@@ -10,14 +11,36 @@
...
-          current-parameters: true
-          predefined-parameters: |
-            TESTHEAD=1
-            cloudsource=develcloud{version}
-            nodenumber=2
-            want_sles12sp2=1
-            mkcloudtarget=all
+      - multijob:
+          name: 'SLE12 SP2 Gate Checks'
+          condition: SUCCESSFUL
```
Comment (jdsn) [inline scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-sle12sp2.yaml]:
```
 -          condition: SUCCESS
+          condition: SUCCESSFUL
```
Is both correct? Or was it broken before?

Comment (dirkmueller) [inline scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-sle12sp2.yaml]: SUCCESSFUL is correct, SUCCESS is not

**Pull request merged**